vck/src/commonMain/kotlin/at/asitplus/wallet/lib/agent/KeyMaterial.kt

```diff
@@ -90,6 +93,18 @@ class EphemeralKeyWithoutCert(
     override suspend fun getCertificate(): X509Certificate? = null
 }
 
+/**
+ * Generate new key material with a random key, e.g. used in tests
+ */
+class KeyWithoutCert(
```

Comment:
Isn't there something like `EphemeralKeyWithoutCert()` already?

Reply:
IIRC the inheritance hierarchy was problematic. I'll re-check

Reply:
gone